bin/resources/GameIndex.yaml

```diff
@@ -2130,51 +2140,82 @@ SCCS-40005:
     halfPixelOffset: 1 # Fixes effect misalignment.
     moveHandler: "MV_Ico" # Fixes depth buffer post-processing.
 SCCS-40006:
-  name: "Zhen Sanguo Wushuang 2"
+  name: 真·三国無双2
+  name-sort: "Zhen - Sanguo Wushuang 2"
+  name-en: "Zhen Sanguo Wushuang 2"
   region: "NTSC-C"
 SCCS-40007:
-  name: "Arc the Lad - Seirei no Tasogare"
+  name: 亚克传承　～精灵之黄昏～
+  name-sort: "Yake Chuancheng - Jingling Zhi Huanghun -"
+  nam-ene: "Arc the Lad - Seirei no Tasogare"
   region: "NTSC-C"
 SCCS-40009:
-  name: "Dragon Ball Z 2"
+  name: 龙珠二世
+  name-sort: "Longzhu Ershi"
+  name-en: "Dragon Ball Z 2"
   region: "NTSC-C"
 SCCS-40010:
-  name: "Super Puzzle Bobble 2"
+  name: 超级泡泡龙 2
+  name-sort: "Chaoji Paopaolong 2"
+  nam-en: "Super Puzzle Bobble 2"
   region: "NTSC-C"
 SCCS-40011:
-  name: "Armored Core 2 - Another Age"
+  name: 机战佣兵2
+  name-sort: "Jizhan Yongbing 2"
+  name-en: "Armored Core 2 - Another Age"
+  region: "NTSC-C"
+SCCS-40012:
+  name: 格斗擂台4 进化版
+  name-sort: "Gedou Leitai 4 Jinhuaban"
+  name-en: "Virtua Fighter 4 Evolution"
   region: "NTSC-C"
 SCCS-40014:
-  name: "World Soccer Winning Eleven 7 - International"
+  name: 实况足球 胜利十一人7 国际版
+  name-sort: "Shikuang Zuqiu Shengli Shiyi Ren 7 Guojiban"
+  name-en: "World Soccer Winning Eleven 7 - International"
   region: "NTSC-C"
 SCCS-40015:
-  name: "Viorate no Atelier - Gramnad no Renkinjutsushi 2"
+  name: 薇欧蕾特的工房 ～古拉姆那特的炼金术士2～
+  name-sort: "Wei'ouleite De Gongfang - Gulamunate De Lianjinshushi 2 -"
+  name-en: "Viorate no Atelier - Gramnad no Renkinjutsushi 2"
   region: "NTSC-C"
 SCCS-40016:
-  name: "Ape Escape - Pumped & Primed"
+  name: 比波猴百斗派对
+  name-sort: "Bibohou Baidou Paidui"
+  name-en: "Ape Escape - Pumped & Primed"
   region: "NTSC-C"
   gsHWFixes:
     halfPixelOffset: 2 # Corrects fullscreen bloom misalignment.
 SCCS-40017:
-  name: "EyeToy - Play"
+  name: 爱淘儿
+  name-sort: "AiTao'er"
+  name-en: "EyeToy - Play"
   region: "NTSC-C"
   compat: 5
 SCCS-40018:
-  name: "Saru EyeToy - Oosawagi! Ukkiuki Game Tenkomori!!"
+  name: 比波猴@爱淘儿 一起欢聚比波猴乐园吧！
+  name-sort: "Bibohou Aitao'er Yiqi Huanju Bibohou Leyuan Ba!"
+  name-en: "Saru EyeToy - Oosawagi! Ukkiuki Game Tenkomori!!"
```

Comment:
EyeToy - Monkey Mania

Reply:
I don't think it should be, it should be what it was originally in the gamedb, which is what they've done.